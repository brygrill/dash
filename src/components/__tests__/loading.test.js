import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import LoadingComponent from '../loading';

describe('<LoadingComponent />', () => {
  it('Renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<LoadingComponent />, div);
  });
  it('Matches snapshot', () => {
    const component = renderer.create(<LoadingComponent />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
