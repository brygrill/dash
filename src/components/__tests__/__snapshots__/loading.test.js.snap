exports[`<LoadingComponent /> Matches snapshot 1`] = `null`;
